flask_cors/__init__.py

```diff
@@ -544,3 +561,12 @@ def _serialize_options(options):
 
     if isinstance(options.get('max_age'), timedelta):
         options['max_age'] = str(int(options['max_age'].total_seconds()))
+
+
+def getLogger(app=None):
```

Comment:
One thing that bothers me with current approach is referencing variables that on one hand are marked as protected/internal but on the other are mentioned in Flask doc (althought it's usage is discouraged).